curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/655727b2e1e49d6adf584442.md

```diff
@@ -0,0 +1,770 @@
+---
+id: 655727b2e1e49d6adf584442
+title: Step 64
```

Comment:
Also for this step, this is the first time we are introducing `createElement()` method and `createTextNode()` method so brief explanations would be good here

You might consider also breaking this up into two steps.